src/test/java/mafia/mafiatogether/lobby/application/dto/response/LobbyInfoResponseTest.java

```diff
@@ -0,0 +1,34 @@
+package mafia.mafiatogether.lobby.application.dto.response;
+
+import mafia.mafiatogether.lobby.domain.Lobby;
+import mafia.mafiatogether.lobby.domain.LobbyInfo;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LobbyInfoResponseTest {
```

Comment:
```suggestion
@SuppressWarnings("NonAsciiCharacters")
class LobbyInfoResponseTest {
```
dto 테스트를 하시게된 이유가 뭔가요?

Reply:
lobby, name을 받아서 조금의 stream 연산을 행하기 때문에 했던 것 같아요. 사실 있어도 그만 안해도 그만인데, 테스트는 다다익선이라고 생각해서요